server/app/services/program/ProgramDefinition.java

```diff
@@ -751,6 +756,9 @@ public abstract Builder setLocalizedConfirmationMessage(
 
     public abstract Builder setAcls(ProgramAcls programAcls);
 
+    public abstract Builder setLocalizedSummaryImageDescription(
```

Comment:
question/suggestion (blocking): Do you need an `addLocalizedSummaryImageDescription` method so that translations will be saved?

Reply:
The `add` methods that exist for the name, description, and confirmation message are only used in tests for easy program creation. I just used the `set` method in tests, but I can also write an `add` method to keep things consistent if that's preferred!